parentchain/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "pallet-parentchain"
+description = "The remote attestation registry and verification pallet for integritee blockchains and parachains"
+version = "0.9.0"
+authors = ["Integritee AG"]
+homepage = "https://integritee.network/"
+repository = "https://github.com/integritee-network/pallet-teerex/"
+license = "Apache-2.0"
+edition = "2018"
+resolver = "2"
+
+[dependencies]
+serde = { features = ['derive'], optional = true, version = '1.0.101' }
+log = { version = "0.4.14", default-features = false }
+hex-literal = { version = "0.3.2", optional = true }
+
+codec = { default-features = false, features = ['derive'], package = 'parity-scale-codec', version = '2.0.0' }
+frame-support = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
+frame-system = { default-features = false, package = 'frame-system', git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
+sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
+sp-io = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
+sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
+sp-std = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
+frame-benchmarking = { default-features = false, optional = true, git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
+sp-keyring = { git = "https://github.com/paritytech/substrate.git", branch = "master", version = "4.0.0-dev" }
```

Comment:
Now it's muuuuch nicer to look at. And easier to review, which explains why I only just now noticed: You're importing std dependent libraries here, which will not work in the sgx-runtime. It needs to be std free.
So either make it optional and import it only in case the std feature is activated or import it with
`default-features = false`.

For keyring, no no_std import is possible : https://github.com/paritytech/substrate/blob/master/primitives/keyring/Cargo.toml

For pallet-balances there is: https://github.com/paritytech/substrate/blob/master/frame/balances/Cargo.toml#L30-L40

Reply:
now the question is: What do you need these libaries for? If you need them for testing purpose only, import them as dev-dependencies: https://github.com/integritee-network/worker/blob/master/core-primitives/types/Cargo.toml#L39-L40

Reply:
dev-dependencies ❤️ 
And then wedo not need to set the std-feature?

Reply:
No. cargo test is std - only anyway and dev-deps will not be reimported.